OnProfileUpdate/function.json

```diff
@@ -0,0 +1,16 @@
+{
+  "bindings": [
+    {
+      "type": "cosmosDBTrigger",
+      "name": "documents",
+      "direction": "in",
+      "connectionStringSetting": "CosmosDbConnectionString",
+      "databaseName": "%COSMOSDB_NAME%",
+      "collectionName": "profiles",
+      "startFromBeginning": true,
+      "createLeaseCollectionIfNotExists": true,
+      "leaseCollectionName": "profile-emails-leases"
+    }
+  ],
+  "scriptFile": "../dist/OnProfileUpdate/index.js"
+}
```

Comment:
To address transient errors effectively, it's essential to incorporate a retry block within the trigger. This implementation will enable the trigger to automatically retry execution in the event of such errors. See [here](https://github.com/pagopa/io-functions-elt/blob/d360d3ac1e560232b9be0dea5448c5956b76d266/CosmosApiMessagesChangeFeed/function.json#L17)